### PR TITLE
zoom: Update to 5.17.0.28375 and modify the manifest

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [x] ♻️ Refactor
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other
## Description
Updating to the latest version
Updating the manifest and use a new approach to grab the version
## Related Tickets & Documents
None
## [optional] What gif/emoji best describes this PR or how it makes you feel?
:alien: :zap: :wrench:
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Notes in the manifest?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Dependencies?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Application Suggestions?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### File persistence to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### zoom

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
